inspec/bigip-ready/controls/example.rb

```diff
@@ -4,24 +4,33 @@
 
 
 # load data from Terraform output
-# created by terraform output --json > bigip/files/terraform.json
+# created by terraform output --json > inspec/bigip-ready/files/terraform.json
 content = inspec.profile.file("terraform.json")
 params = JSON.parse(content)
 
-BIGIP_DNS  = params['bigip_mgmt_public_ips']['value']
-BIGIP_PORT = params['bigip_mgmt_port']['value']
+BIGIP_DNS       = params['bigip_mgmt_public_ips']['value']
+BIGIP_PORT      = params['bigip_mgmt_port']['value']
+BIGIP_PASSWORD  = params['bigip_password']['value']
 
-control "Connectivity" do
+control "AS3 Connectivity" do
   impact 1.0
-  title "BIGIP is reachable"
+  title "BIGIP is ready and reachable"
 
   BIGIP_DNS.each do |bigip_host|
     describe host(bigip_host, port: BIGIP_PORT, protocol: 'tcp') do
         it { should be_reachable }
     end
-    # TODO: include testing of authentication against API
+    testurl = 'https://' + bigip_host + ':' + BIGIP_PORT + '/mgmt/shared/appsvcs/info'
```

Comment:
Maybe use string interpolation here? http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html

Reply:
I've committed the change. I haven't tested it yet.